package.json

```diff
@@ -2,7 +2,7 @@
   "name": "com.amazonaws.gamelift",
   "displayName": "Amazon GameLift Plugin for Unity",
   "author": "Amazon.com, Inc.",
-  "version": "1.3.1",
+  "version": "1.4.0",
```

Comment:
Thanks for making this change Kyle! Is this for the final release or for bug tracking purposes during development? The version number for release is 2.0.0, so if this is meant for launch we should update this line and the commit message.